scheduler/util.go

```diff
@@ -414,6 +406,26 @@ func tasksUpdated(jobA, jobB *structs.Job, taskGroup string) bool {
 	return false
 }
 
+func networkUpdated(netA, netB []*structs.NetworkResource) bool {
```

Comment:
Huh, I wonder why this doesn't use https://godoc.org/github.com/hashicorp/nomad/nomad/structs#Networks.Equals